src/nr_openai_observability/bedrock.py

```diff
@@ -314,9 +319,12 @@ def build_bedrock_events(response, event_dict, time_delta):
 
         if len(messages) > 0:
             messages[-1]["is_final_response"] = True
+            ai_message_id = create_ai_message_id(messages[-1].get("id"))
+            set_ai_message_ids([ai_message_id])
 
         summary = {
             "id": completion_id,
+            "conversation_id": get_conversation_id(),
```

Comment:
There should be at least one other spot to add conversation Id. It looks like it is being added to messages as well and if so, there is a method in bedrock.py where this will need to be added.

Reply:
Good catch. It looks like I'm also not currently capturing the model for the final message call in langchain when bedrock is used too.